src/contribute/release-guide.md

```diff
@@ -150,6 +150,19 @@ You should verify that the issues listed automatically by JIRA are appropriate t
 
 Adjust any of the above properties to the improve clarity and presentation of the Release Notes.
 
+### Check the state of the aggregate javadoc
+
+When you build with -Prelease, the Maven build process creates a single archive of javadoc for all of
+the Java API. This takes place in `sdks/java/javadoc`. The actual build of the javadoc is run from
+Apache Ant, via the `maven-antrun-plugin`. 
+
+The javadoc requires some maintenance; if you look at the `ant.xml` file, you will see that it 
+has a list of 'offline' URLs and a list of excluded packages. The offline links connect the Beam
+javadoc to the javadoc of other projects such as Jackson and Guava. Check that the URLs for the offline
+project point to the current versions.
+
+Also check that the excluded package list is up to date. 
```

Comment:
I would make this more direct/prescriptive with less prose if possible. Here's a minor pass, but feel free to use in your own voice (or tell me to do it myself in fixup :).

Title: `Update and verify Javadoc`

The `sdks/java/javadoc/ant.xml` file contains a list of modules to include in and exclude from the released javadoc, plus a list of offline URLs that populate links to Javadoc for other modules that Beam depends on.

Confirm that new modules added since the last release have been added to the inclusion list as appropriate.

Confirm that the excluded package list is up to date.

Verify the version numbers for offline links match the versions used by Beam. If the version number has changed, download a new version of the corresponding `XX-docs/package-list` file.